### PR TITLE
Anchorpoint ERT Station

### PR DESCRIPTION
**Pull request by Steelpoint**
# About the pull request

Adds a ERT station for CMB aligned ERT's to return to, or for admin events. The station contains a CMB fax. Anchorpoint is designed to appear larger than it actually is. 

In-game image in screenshot section.

# Explain why it's good for the game

Gives more love to the CMB so they have somewhere official to land. Can also be used for setting up ERTs for events or for holding prisioners. 

Also has a CMB fax for roleplay oppertunities. 

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

anchorpoint

</details>

# Changelog
:cl:
mapadd: Anchorpoint has been added as a CMB ERT station. CMB aligned ERT's will return to this station when leaving the USS Almayer. The station also contained a CMB fax machine. 
/:cl:

### CODE REVIEW

**Comment by Steelpoint** [comment]
I was going to have the station contain three landing pads, so multiple ships can land on the same ERT station. However, the game will only ever use 1 of the landing pads when someone leaves on a CMB shuttle, meaning multiple Anchorpoints will be spawned for every CMB shuttle that leave. 

So I've only made one of the landing pads work.

File: maps/templates/lazy_templates/anchorpoint_ert_station.dmm
Comment (Nanu308) [inline maps/templates/lazy_templates/anchorpoint_ert_station.dmm:1]: Requires more work, not up to standard compared to the other ERT stations.

**Review by Nanu308 — CHANGES_REQUESTED** [review]

File: maps/templates/lazy_templates/anchorpoint_ert_station.dmm
Comment (Steelpoint) [reply maps/templates/lazy_templates/anchorpoint_ert_station.dmm:1]: In what way? More rooms? More details? Bigger? Smaller? In comparison to some of the ERT stations this map meets similar standards of size, contents, detailing and room.

**Comment by Steelpoint** [comment]
I've gone ahead and made some cosmetic changes. I'm not really sure exactly what is not "up to standard" so I'll either leave this as it stands or wait for further direction.

**Comment by LC4492** [comment]
I suppose you should make a more complex containment site, maybe even with a smaller-sized court. It also should look more CMB-oriented, try looking at Hybrisa's CMB Local Department for inspiration.

**Pull request closed without merging**